dts/bindings/pwm/renesas,rcar-pwm.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2022, IoT.bzh
+# SPDX-License-Identifier: Apache-2.0
+
+description: Renesas R-Car PWM controller
+
+compatible: "renesas,rcar-pwm"
```

Comment:
just noticed that Linux is using 
"renesas,pwm-rcar"
https://elixir.bootlin.com/linux/v5.17-rc3/source/Documentation/devicetree/bindings/pwm/renesas,pwm-rcar.yaml
Please renease this file and the compatible entry.

Reply:
Corrected. Now declared like Linux dts files.